qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/expression/Parts.java

```diff
@@ -78,10 +78,42 @@ public NamespacePart getNamespacePart() {
 		}
 		return null;
 	}
-	
+
+	/**
+	 * Returns the property part of the parts and null otherwise.
+	 *
+	 * @return the property part of the parts and null otherwise.
+	 */
+	public PropertyPart getPropertyPart() {
+		if (super.getChildCount() == 0) {
+			return null;
+		}
+		Part secondPart = getChild(1);
+		if (secondPart.getPartKind() == PartKind.Property) {
+			return (PropertyPart) secondPart;
+		}
+		return null;
+	}
+
+	/**
+	 * Returns the method part of the parts and null otherwise.
+	 *
+	 * @return the method part of the parts and null otherwise.
+	 */
+	public MethodPart getMethodPart() {

```

Comment:
Same thing it is wrong because you get the first method part and not other lethod part